pss/pss_test.go

```diff
@@ -48,28 +50,112 @@ func TestTrojanChunkRetrieval(t *testing.T) {
 	payload := []byte("RECOVERY CHUNK")
 	topic := trojan.NewTopic("RECOVERY")
 
-	var ch chunk.Chunk
-
 	// call Send to store trojan chunk in localstore
-	if ch, err = pss.Send(ctx, testTargets, topic, payload); err != nil {
+	if _, err = pss.Send(ctx, targets, topic, payload); err != nil {
 		t.Fatal(err)
 	}
 
+	var chunkAddress chunk.Address
+	for po := uint8(0); po <= chunk.MaxPO; po++ {
+		last, err := localStore.LastPullSubscriptionBinID(po)
+		if err != nil {
+			t.Fatal(err)
+		}
+		if last == 0 {
+			continue
+		}
+		// iter for chunk in localstore
+		ch, _ := localStore.SubscribePull(context.Background(), po, 0, last)
+		for c := range ch {
+			chunkAddress = c.Address
+			break
+		}
+	}
```

Comment:
i suppose this is a way to get the last sent chunk address? is this correct?

Reply:
Correct, this is exactly its purpose.

Since the `chunk stored` is no longer returned in the `Send` method, and localstore does not give visibility into its content, we must iterate over it in order to get what chunk was stored.

Reply:
I added a comment in the code for future reference.

Reply:
excellent! thank you very much